### PR TITLE
Remove ini

### PR DESCRIPTION
**Pull request by sethrosetter**
- 'provisioning_config' is now the source of truth in 
   - provisioning 
   - funtional test framework
   - gatling / gateload config generation

### CODE REVIEW

**Comment by sethrosetter** [comment]
@ashvindersingh please take a look at this as well

File: lib/syncgateway.py [hunk 1]
```diff
@@ -23,3 +23,6 @@ def start(self, config):
 
     def restart(self, config):
         orchestration.syncgatewayactions.restart(self.host_name, config)
+
+    def __str__(self):
```
Comment (ashvindersingh) [inline lib/syncgateway.py]: If this implemented for debugging purposes then I would recommend using __repr__

File: lib/cluster.py [hunk 2]
```diff
@@ -1,20 +1,66 @@
...
+            sys.exit(1)
+
+        i = ansible.inventory.Inventory(host_list=hostfile)
+        hosts = i.get_group(tag).get_hosts()
+        return [host.get_variables() for host in hosts]
 
     def reset(self, config):
         clusteractions.reset(config)
 
+    def __str__(self):
```
Comment (ashvindersingh) [inline lib/cluster.py]: __repr__ will give more accurate description of the object

**Comment by sethrosetter** [comment]
@ashvindersingh updated per your request

**Comment by ashvindersingh** [comment]
Thanks!

**Pull request merged**